### PR TITLE
Fix for the Jira Issue : EMM-1008

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/emm/src/emm/units/header/public/js/header.js [hunk 1]
```diff
@@ -64,9 +64,14 @@ $(document).ready(function () {
...
                             $(errorMsgWrapper).removeClass("hidden");
                         }
-                    }, function () {
-                        $(errorMsg).text("An unexpected error occurred. Please try again later.");
-                        $(errorMsgWrapper).removeClass("hidden");
+                    }, function (data) {
+                        if (data.statusCode == 400) {
+                            $(errorMsg).text("Old password does not match with the entered value. Please check.");
+                            $(errorMsgWrapper).removeClass("hidden");
+                        }else {
```
Comment (harshanL) [inline modules/apps/jaggery/emm/src/emm/units/header/public/js/header.js]: Check formatting here.

Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/header/public/js/header.js]: These formatting issues has been fixed in a recent PR

**Comment by dakshika** [comment]
I have notice many  console.log on js. Why do we need to push them to final code?

**Comment by Kamidu** [comment]
These method identified as send system related operations to usage which does need to be notify to the end user when the introduction of backend invoker .
But in the PR[1] the above mentioned console logs were translated to User interface appropriately. 

[1]. #166

**Pull request merged**